### ISSUE TITLE
lack Chinese aircraft crew，Please bother the developer or author to update it.

### ISSUE THREAD

**Issue by Momoyih** [GaijinEntertainment/fmod_studio_warthunder_for_modders#159]

**Comment by Uchuujin01** [comment]
Hello. I hope will find spare time and will do it later, but, you can add any crew banks and files by yourself, if you planning to create crew voice mod.

**Comment by Uchuujin01** [comment]
I mean, the list of files is the same for every language.

**Comment by Momoyih** [comment]
> 你好。我希望能找到空闲时间，以后再做，但是，如果你打算创建船员语音模块，你可以自己添加任何船员库和文件。

No, no, no, radio voice, such as attacking point A, point B and point C, I don't know what to do. After being exported, it is still the original sound in the game.

**Comment by Uchuujin01** [comment]
We have those message sound events in fmod project presented here. It is crew_dialogs_common sound bank ( placed in dialogs_crew_aircraft/ fmod folder). You can add your disired language sound bank, call it _crew_dialogs_common_zh or _crew_dialogs_common_zhhx, and add there an audio table with your files. The main thing - they should be called like the files from another language pack, English for example.

**Comment by Uchuujin01** [comment]
Don't forget to switch on sound mods in your config.blk file BTW.
